doc/source/performance/aio_page_requests.rst

```diff
@@ -34,7 +35,16 @@ Status Variables
   :vartype: Numeric
   :scope: Global
 
-This variable shows the number of submitted buffered asynchronous I/O requests.
+This variable shows the number of asynchronous I/O requests buffered in the
+Linux Native subsystem before submission.
```

Comment:
I don't understand this change?

Looking at the status variable, it only increases at https://github.com/percona/percona-server/blob/5.7/storage/innobase/os/os0file.cc#L3103, and never decreases.

So this variable is updated after the request was submitted to the kernel, not before.
Also, it is an always increasing total submitted counter, not the number of entries we currently have in the buffer, which is suggested by this change.

Reply:
Got it. Thanks.